python/sdk/xain_sdk/__init__.py

```diff
@@ -4,21 +4,20 @@
 
 from .interfaces import TrainingInputABC, TrainingResultABC
 from .participant import ParticipantABC, ParticipantError
+from .utils import configure_logging  # pylint: disable=import-outside-toplevel
```

Comment:
In theory the `pylint` hint could be removed now.

Reply:
done